modules/developer_manual/pages/core/apis/ocs-share-api.adoc

```diff
@@ -676,8 +676,11 @@ The permissions to set on the share.
 * 2 = update;
 * 4 = create;
 * 8 = delete;
-* 15 = read/write;
 * 16 = share;
+
+Common example combinations are:
+
+* 15 = read/write(update and create)/delete;
```

Comment:
Hmmm, maybe I got it wrong, but in my calculation:
```
 2 ... update
 4 ... create
 8 ... delete
14 and not 15
```
Additionally `31` (see below) is also not the sum of the upper ones if I am correct with my view...

@jnweiger can you bring some light into it?

Reply:
read = 1, 1+2+4+8=15

another 16 for share gets to 31, the maximum

Reply:
I missed the read = 1 🤦‍♂️